project/Dependencies.scala

```diff
@@ -3,13 +3,13 @@ import sbt._
 object Dependencies {
 
   object Compile {
```

Comment:
In our template projects for services the convention is for the dependencies file to expose the dependencies as ready-made sequences e.g.

```
object AppDependencies {
  val compile = Seq(...)
  val test = Seq(...)
}
```

The `build.sbt` file then simply contains:
```
libraryDependencies ++= AppDependencies.compile ++ AppDependencies.test,
```

Reply:
Marked all dependencies as `Test` and converted into a `Seq` as suggested above.